includes/templates/responsive_classic/css/stylesheet_css_buttons.css

```diff
@@ -1,22 +1,19 @@
-.buttonRow a {text-decoration:none;}
-.button, input.button, input.cssButtonHover {display:inline-block;font-size:1.3em;margin:0;padding:8px 20px;text-decoration:none;}
-input.submit_button {border:none !important;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
-input.submit_button:hover {border:none !important;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px !important;}
-input.cssButtonHover {border:none;cursor: pointer;border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px !important;}
-span.normal_button {}
-span.normal_button:hover {}
-span.cssButton.normal_button.button.button_more_reviews, .button_more_reviews:hover, span.cssButton.normal_button.button.button_read_reviews, .button_read_reviews:hover{display:block;text-align:center;}
-span.cssButton.normal_button.button.button_write_review, .button_write_review, span.cssButton.normal_button.button.button_in_cart{display:block;text-align:center;font-size:130%;padding:12px 20px !important;}
-#indexBody span.cssButton.normal_button.button.button_in_cart, #indexBody .button_in_cart:hover{display:inline-block;}
-span.cssButton.normal_button.button.button_goto_prod_details{background:#000;}
-.button_goto_prod_details:hover{background:#05a5cb !important;}
-input.cssButton.submit_button.button.button_search, .button_search:hover{padding:8px 20px !important;font-size:1.0em;line-height:18px;}
-#advSearchDefault input.cssButton.submit_button.button.button_search, #advSearchDefault .button_search:hover{font-size:1.3em;}
+.buttonRow a { text-decoration: none; }
+.cssButton {display:inline-block;font-size:1.3em;margin:0;padding:8px 20px;text-decoration:none;}
+input.submit_button {border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
+input.submit_button:hover {border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
+input.cssButtonHover {border:none;cursor: pointer;border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
```

Comment:
Did you intend to keep this `input.cssButtonHover` line?

Reply:
I assumed that is template specific. Searching for cssButtonHover does not show up anything though so it may not be used. The html_output had nothing that created that class. I'm sure a ton of the css in that template could be cleaned up.

Reply:
> The html_output had nothing that created that class.

Hmmm. I mentioned it because I saw that you **had** deleted it from html_output:
<img width="598" alt="Image 2020-04-18 at 3 45 32 PM" src="https://user-images.githubusercontent.com/404472/79669749-cc861d00-818b-11ea-9027-f562fb06a9cb.png">